### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by mrosso10**
# Summary
* Implemented blog app from Rails tutorial

# Test plan
1. Go to the home page
2. Click on the 'New article' link
3. Login with username 'dhh' and password 'secret'
4. Create a new article
5. Go to /articles
6. Click on the 'Edit' link and update the created article
7. Create a new comment for the article
8. Go back to /articles and destroy the article

# Trello Card
https://trello.com/c/6f7onHIE/22-crear-blog-base

### CODE REVIEW

**Comment by mrosso10** [comment]
#1

File: app/views/articles/show.html.slim [hunk 1]
```diff
@@ -0,0 +1,17 @@
+p
+  strong> Title
```
Comment (Dinuuu) [inline app/views/articles/show.html.slim]: is the `>` intended ?

File: app/views/articles/show.html.slim [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  = @article.title
+
+p
+  strong> Text
```
Comment (Dinuuu) [inline app/views/articles/show.html.slim]: idem

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (Dinuuu) [inline app/views/layouts/application.html.erb]: This should be changed to a Slim file to maintain concistency

**Comment by Dinuuu** [comment]
Small comments and ready to go :+1:

File: app/views/articles/show.html.slim (hunk 1 above)
Comment (mrosso10) [inline app/views/articles/show.html.slim]: yes

File: app/views/articles/show.html.slim (hunk 2 above)
Comment (mrosso10) [inline app/views/articles/show.html.slim]: yes

**Pull request merged**